course/__init__.py

```diff
@@ -0,0 +1,4 @@
+import pandas as pd
+
+test_data = pd.read_json("./dumps/courseData.json");
+test_data = {k: val.groupby('pk')['fields'].apply(list).apply(lambda x: x[0]).to_dict() for k, val in test_data.groupby("model")};
```

Comment:
this piece of code is hard to read, it's unclear what it does.

besides, why is anything here in this file? That smells fishy